src/features/annotation-thread/AnnotationThread.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @flow
+ * @file Annotation Thread Container
+ * @author Box
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import AnnotationThreadContent from './AnnotationThreadContent';
+import API from '../../api/APIFactory';
+import APIContext from '../../elements/common/api-context';
+import Internationalize from '../../elements/common/Internationalize';
+import { DEFAULT_HOSTNAME_API, ORIGIN_ANNOTATION_THREAD } from '../../constants';
+import { withErrorBoundary } from '../../elements/common/error-boundary';
```

Comment:
I would suggest adding an error boundary in the consuming application rather than at the component level.